specs/core/0_beacon-chain.md

```diff
@@ -1631,8 +1631,15 @@ For each `attestation` in `block.body.attestations`:
 ```python
     assert attestation.custody_bitfield == b'\x00' * len(attestation.custody_bitfield)  # [TO BE REMOVED IN PHASE 1]
     assert attestation.aggregation_bitfield != b'\x00' * len(attestation.aggregation_bitfield)
-
-    for i in range(len(crosslink_committee)):
+    
+    crosslink_committee = [
+        committee for committee, shard in get_crosslink_committees_at_slot(state, attestation.data.slot)
+        if shard == attestation.data.shard
+    ][0]
+    verify_bitfield(attestation.aggregation_bitfield, len(crosslink_committee))
+    verify_bitfield(attestation.custody_bitfield, len(crosslink_committee))
+    
+    for i in range(len(crosslink_committee):
```

Comment:
@JustinDrake Is this for loop necessary? Aren't we already checking this using 
`assert attestation.custody_bitfield == b'\x00' * len(attestation.custody_bitfield)`?

Reply:
I have the same question here. Can we remove this for-loop? @djrtwo @JustinDrake

Reply:
Oh I got it, because `assert attestation.custody_bitfield == b'\x00' * len(attestation.custody_bitfield)` is only valid in Phase 0.